src/main/java/org/traccar/sms/smpp/SmppClient.java

```diff
@@ -44,6 +31,17 @@
 import com.cloudhopper.smpp.type.SmppChannelException;
 import com.cloudhopper.smpp.type.SmppTimeoutException;
 import com.cloudhopper.smpp.type.UnrecoverablePduException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.traccar.Context;
+import org.traccar.notification.MessageException;
+import org.traccar.sms.SmsManager;
+
+import java.util.concurrent.ExecutorService;
```

Comment:
Please don't rearrange imports. It might cause merge conflicts later on.

Reply:
Fixed it!